<launch>

  <node pkg = "necst_ros3"
      name = "hot_position"
      type = "hot_position.py" />

  <node pkg = "necst_ros3"
      name = "hot_position_sim"
      type = "hot_position_sim.py" />

  <node pkg = "necst_ros3"
      name = "hot_position_lock"
      type = "hot_position_lock.py" />

</launch>
